TeamMandate/Presentation.md

```diff
@@ -10,112 +10,80 @@ As a student who isn't taking any courses this summer, I decided to look for wor
 
 ... 
 
-## The How
+## Proposed IITB Mandated Priorities
 
-- Do not compete with private sector 
-  - Integration over development 
-  - Create jobs, not software 
+### Avoid Rework 
 
->delivering programs that help Canadians move through life’s transitions, from school to work, from one job to another, *from unemployment to employment* and from the workforce to retirement; 
+Work in the open
 
-- Avoid rework 
-  - Work in the open 
+#### Repercussions
+
+- All teams (Ex: developers, security and testing) must use git
+
+#### Opportunities
```

Comment:
I'm interpreting opportunities as tangible avenue where we can start focusing on. E.g.

- Automate testing, security compliance, and deployment releases
- Publish ESDC source code under an open source licence, accessible on the Internet, and use crowd sourcing to fix issues
- Leverage SaaS for collaboration with unclassified information. Using government resources (i.e. local network) only for sensitive information
- Aggressively move workloads to the public cloud. New workloads to be build and deployed to the public cloud.
- Enable remote access to emails and corporate systems (e.g. HR, Finance). Remote access from personal devices.
- Re-evaluate IITB's risk management approach. From a control-based one to a monitor and resilience-based one.
- Support IT Security and Architects towards a mentorship role instead of assessor role